packages/core/src/components/TextArea/index.tsx

```diff
@@ -30,9 +32,11 @@ export default class TextArea extends React.Component<Props, State> {
   };
 
   static defaultProps = {
+    alwaysHighlight: false,
     locale: 'none',
     noTranslate: false,
     proofread: false,
+    secondaryMark: false,
```

Comment:
This seems weird. So the consumer toggles between the colors manually? It's not automated?